nh_viral
Quantifying viral loads in Apis mellifera and Ceratina calcarata in various landscapes

Viral RNA was extracted from whole bees and sequenced on an Illumina NovaSeq6000, paired end reads. 

Read quality was checked with FastQC

Reads mapped using STAR aligner 2.7.10b using default parameters, with the exception of the --genomeSAindexNbases function, which was set to 5. This was done to accomodate the small viral reference genomes. 
